.rubocop.yml

```diff
@@ -52,5 +39,14 @@ Metrics/BlockLength:
     - '**/*.rake'
     - 'spec/**/*.rb'
 
+Metrics/MethodLength:
+  Exclude:
+    - 'spec/support/omniauth_macros.rb'
+    - 'app/presenters/nav_presenter.rb'
+
 Rails/SkipsModelValidations:
   Enabled: false
+
+Rails/LexicallyScopedActionFilter:
+  Exclude:
+    - 'app/controllers/application_controller.rb'
```

Comment:
We could just disable locally? 
e.g.in nav_presenter.rb
def navs # rubocop:disable Metrics/MethodLength
...